_posts/2023-03-02-q1-survey-results.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+authors:
+  - apasun
+title: "OpenSearch Receives High Marks in the Quarterly Q1 Community Survey"
+date: 2023-03-02 
+categories:
+- community
+
```

Comment:
@hdhalter - Please add the meta below: 

**meta_keywords:** OpenSearch NPS, OpenSearch net promoter score, OpenSearch user experience 
**meta_description:** OpenSearch receives high marks in its quarterly Q1 Community Survey, reporting a net promoter score of 57.8 and a UX health score of 77.2.

Reply:
Done.

Reply:
@pajuric - how do you enter an author bio?

Reply:
Sure, I can do that.

Reply:
Added in #1419